lib/exercism/active_record_citext.rb

```diff
@@ -1,4 +0,0 @@
-# from http://gray.fm/2013/09/17/unknown-oid-with-rails-and-postgresql/
-ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.tap do |klass|
-  klass::OID.register_type('citext', klass::OID::Identity.new)
-end
```

Comment:
I got errors due to this file. Removing it passed all the tests. I am not able to open the link mentioned here - http://gray.fm/2013/09/17/unknown-oid-with-rails-and-postgresql/. Anybody knows the reason why it was added?